.gitignore

```diff
@@ -29,6 +29,7 @@ app/api/csv/specs/*.jpg
 app/api/attachments/specs/uploads/toDelete.txt
 uploaded_documents/*
 custom_uploads/*
+activity_log/*
```

Comment:
Why adding a new folder when we already have a `log` folder that is already configured in the paths for production? I don't think this is a bad idea, at the same time this forces our production code and self-installed instances to manage this new variable when we release this code.

Reply:
Given the activity log files include in its name the tenant I'll write them into the log folder.